### DISCUSSION TITLE
Dev versions

### DISCUSSION THREAD

**Discussion by davidlas** [nasko222/the-minecraft-recreation-project#152] [Q&A]
Is there somewhere that you have a timeline or changelog for the developer versions (Or unreleased  for indev/infdev)?

**Comment by nasko222** [comment]
I don't actually have it nearby, there's videos by TheMisterEpic and few other small youtubers like Christo18 too. Also some developer builds are listen on the mc wiki.

**Reply by davidlas** [reply]
So, you couldn't tell me what differences there are between let's say the beta 1.8 dev builds? Those are the main one's I'm interested in.

**Comment by nasko222** [comment]
the beta 1.8 dev builds might have revamp soon but heres somewhat shortened description

dev1 - panorama test
dev2 - stronghold generation test (on the surface)
dev3 - villages, mineshafts, mushrooms, biome gen revamp 1
dev4 - few tweaks to the generation and few more new items
dev5 - height and hunger test
dev6 - enderman test 1
dev7 - enderman test 2
dev8 - coding with notch 1
dev9 - coding with notch 2
dev10 - new villages test
preview - pax build

**Reply by davidlas** [reply]
Thank you. I've wanted to make a polyworld , as you might imagine. One other question. What's the diffrnce in c0.0.2 dev-1?

**Reply by nasko222** [reply]
FPS Counter

**Reply by G23M08** [reply]
When will the builds be done

**Reply by nasko222** [reply]
Staggo is taking an indefinite break and probably wont be very soon but theyre not cancelled. The old b1.8 devs can be downloaded from old snapshot of the repo

**Reply by G23M08** [reply]
What about the other removed recreations

**Reply by nasko222** [reply]
Like?

**Reply by G23M08** [reply]
b1.7 devs old  highly-speculative-mc-builds

**Reply by nasko222** [reply]
highly speculative builds are not recreations. I regret ever making them. They are deleted for good, I want everyone to forget about them

**Reply by G23M08** [reply]
What were them at least also still 1.7 old devs

**Comment by davidlas** [comment]
Oh, and what's the underlying biome code for beta? Is it 1,7 or 1,8?

**Reply by nasko222** [reply]
Not sure what you're refering to

**Comment by davidlas** [comment]
Do you care if the old repository builds of b1.8 are used for vids?

**Reply by nasko222** [reply]
its all fine

**Reply by Its-Just-Luci** [reply]
where can beta 1.8 builds be downloaded, I don't see any in the repo?

**Reply by nasko222** [reply]
as i also said in omni, the b1.8 devs are currently being reworked, the one ones can be downloaded if you look at old commit of the repo

**Comment by davidlas** [comment]
Ok thanks

**Comment by davidlas** [comment]
There’s multiple repos with different build variations  in them

**Comment by Illia24** [comment]
Do you will making other dev versions? But newer or something like that

**Reply by Illia24** [reply]
Newer versions i mean

**Reply by nasko222** [reply]
currently no, also post 1.12 mc is extremely hard to mod without fabric and forge

**Comment by Illia24** [comment]
Ok

нд, 5 лист. 2023 р., 07:53 користувач Nasko ***@***.***> пише:

**Comment by bunnybeep** [comment]
hey, _reeaallllly_ stupid question, but what order do the b1.9 builds go in order of like, order of development/features? i am very tired atm lmfao

**Comment by G23M08** [comment]
alpha 1.2.0 dev bu8ilds crash more often then what it was

**Comment by nasko222** [comment]
send crash log

**Reply by G23M08** [reply]
--- BEGIN ERROR REPORT a1dce528 --------
Generated 17/11/23 14:11

Minecraft: Minecraft Alpha v1.2.0
OS: Windows 10 (amd64) version 10.0
Java: 1.8.0_361, Oracle Corporation
VM: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
LWJGL: 2.9.3
OpenGL: Intel(R) HD Graphics 2500 version 4.0.0 - Build 10.18.10.5129, Intel

java.lang.RuntimeException: Failed to check session lock, aborting
	at cu.<init>(cu.java:236)
	at cu.<init>(cu.java:188)
	at cu.<init>(cu.java:105)
	at net.minecraft.client.Minecraft.b(Minecraft.java:1046)
	at ks.c(SourceFile:68)
	at ks.a(SourceFile:54)
	at bn.a(SourceFile:69)
	at bn.e(SourceFile:115)
	at bn.d(SourceFile:103)
	at net.minecraft.client.Minecraft.i(Minecraft.java:873)
	at net.minecraft.client.Minecraft.run(Minecraft.java:456)
	at java.lang.Thread.run(Unknown Source)
--- END ERROR REPORT 88095ca5 ----------

**Reply by G23M08** [reply]
it was (somehow) the intence that i was using

**Comment by Illia24** [comment]
Sorry to bother but how to get old 1.8 beta dev builds?

**Reply by nasko222** [reply]
How old? the more versions link has the latest b1.8 that are now removed because we are working on new ones

Theres also link to 2021 remakes, there are two b1.8 devs there, these are like the oldest (there could be older ones on some discord chat, but these should do the job), everything is basically clashed into those 2 devs there xd

**Reply by Illia24** [reply]
Ok
